src/catalog/database_catalog.cpp

```diff
@@ -368,8 +369,8 @@ bool DatabaseCatalog::CreateColumn(transaction::TransactionContext *const txn, c
   *len_entry = (col.Type() == type::TypeId::VARCHAR || col.Type() == type::TypeId::VARBINARY) ? col.MaxVarlenSize()
                                                                                               : col.AttrSize();
   *notnull_entry = !col.Nullable();
-  *dbin_entry = reinterpret_cast<uintptr_t>(
-      nullptr);  // TODO(Matt) implement deep copy of the abstract expression once #386 is in
+  // TODO(John) implement deep copy of the abstract expression and store the pointer once #386 is in
+  *dbin_entry = nullptr;
```

Comment:
I'm a bit confused, your PR description said we are no longer using the ADBIN column. Do you mean we should deprecate it, or just always set it to nullptr like done here.